tools/config.py

```diff
@@ -3,16 +3,16 @@
 
 # Update with the Meilisearch version TAG you want to build the image with
 
-MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.29.0'
+MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.29.1'
 
 # Update with the custom image name that you want to publish after TESTING
 
-PUBLISH_IMAGE_NAME = 'meilisearch-v0-29-0-ubuntu-2004-lts-build--06-10-2022-14-20-30'
+PUBLISH_IMAGE_NAME = 'meilisearch-v0-29-1-debian-10-build--24-10-2022-17-28-36'
```

Comment:
👀 can you explain the reason of the change?

Reply:
I add a note on the PR description but there is no Changelog